Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
deleted file mode 100644
--- a/.gitignore	(revision c42a689971ea5d1fb32e17665062522920b3a480)
+++ /dev/null	(revision c42a689971ea5d1fb32e17665062522920b3a480)
@@ -1,8 +0,0 @@
-parrainage.pem
-parrainage.pem.pub
-seConnecterAuServeur
-task-definition.json
-/src/backend/client/assets/*
-/src/frontend/node_modules
-
-
